lib/helpers/ast-node-info.js

```diff
@@ -66,6 +66,9 @@ function childrenFor(node) {
     return node.body;
   }
   if (node.type === 'BlockStatement') {
+    if(node.inverse){
+      return node.program.body.concat(node.inverse.body);
```

Comment:
I believe that its possible to only have an inverse (and therefore `node.program.body` is null/undefined).

Perhaps something like this might work:

```js
return [].concat(
  node.program && node.program.body,
  node.inverse && node.inverse.body
).filter(Boolean);
```

Reply:
Do you have an example of this? then I would be able to add that to the test suite.

Reply:
I think this will do it:

```hbs
{{#unless}}
{{/unless}}
```

Reply:
Okay, I will try that tomorrow. 
It looks like it should only be inverse that is nullable here: https://github.com/glimmerjs/glimmer-vm/blob/e8eb555378f2699676cf39a11fc6dc96af847b58/packages/%40glimmer/syntax/lib/types/nodes.ts#L47

Reply:
ya, agreed

I just did a quick test on astexplorer.net and it seems that program is populated there also, so maybe it just isn't possible?

Reply:
I have not been able to produce a BlockStatement without a `program`. I do not this it is possible, which would also make sense when looking at the typescript definition.
But if you still prefer a null check for the possibility that it may sneak in later, then just let me know.

Reply:
Nope, seems fine to me. Thank you for digging into this!